README.md

```diff
@@ -34,42 +25,23 @@ To get a local copy up and running follow these simple example steps.
 
 ### Run tests
 
-### Deployment
+- Linter
 
+### Deployment
 
+Gonzalo Medina
 
 ## Authors
 
 👤 **Author1**
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-## 🤝 Contributing
-
-Contributions, issues, and feature requests are welcome!
-
-Feel free to check the [issues page](../../issues/).
-
-## Show your support
-
-Give a ⭐️ if you like this project!
```

Comment:
- I think it would be better to use a more descriptive README file for your project - please use [this template](https://github.com/microverseinc/readme-template) but remember to customize it to your project. Always think about people with that you share your code: collaborators, code reviewers, programmers, recruiters, or non-technical users who might want to reuse or improve your project.